joomla/joomla-cms
# Composer update

### Summary of Changes

composer update --no-dev

Lock file operations: 0 installs, 4 updates, 0 removals
  - Upgrading joomla/archive (1.1.8 => 1.1.10)
  - Upgrading joomla/filter (1.4.3 df3403c => 1.4.3 ec95176)
  - Upgrading leafo/lessphp (dev-joomla3-php8 1bddadb => dev-joomla3-php8 a149187)
  - Upgrading psr/log (1.1.3 => 1.1.4)

### Testing Instructions

Make sure the following packages still work:
- joomla/filter
- lessphp (generation of less files from within the template manager)
- joomla/archive (upload + install an extension)
- psr/log

### Actual result BEFORE applying this Pull Request

non up-to-date packages

### Expected result AFTER applying this Pull Request

up-to-date packages

### Documentation Changes Required

none

@brianteeman that should look better now.
